res/css/views/rooms/_RoomSublist2.scss

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import "../../../../node_modules/react-resizable/css/styles.css";
+
+.mx_RoomList2 .mx_RoomSubList_labelContainer {
+    z-index: 12;
```

Comment:
this arbitrary decision is fixed in a later PR - please ignore this particular style choice.